rfb/rfbclient.h

```diff
@@ -373,6 +370,18 @@ typedef struct _rfbClient {
 	LockWriteToTLSProc LockWriteToTLS;
 	UnlockWriteToTLSProc UnlockWriteToTLS;
 
+        /* Hooks for custom rendering
```

Comment:
Sry, I should've mentioned that we use doxygen --> https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html So you have to start comments with /**